tools/flang2/utils/upper/CMakeLists.txt

```diff
@@ -12,12 +12,21 @@ add_executable(upperl
 
 # Generate upper tables
 
+if(${ARCHNAME} STREQUAL "ppc64le")
+add_custom_command(
+  OUTPUT ${UTILS_UPPER_BIN_DIR}/upperilm.h
+  COMMAND LC_ALL=C sort ${UTILS_UPPER_DIR}/upperilm_ppc64le.in | grep -v "^ *\#" > ${UTILS_UPPER_BIN_DIR}/upperilm.sort
+  COMMAND ${CMAKE_BINARY_DIR}/bin/upperl ${UTILS_UPPER_BIN_DIR}/upperilm.sort ${UTILS_UPPER_BIN_DIR}/upperilm.h
+  DEPENDS upperl ${UTILS_UPPER_DIR}/upperilm.in
+  )
+else()
```

Comment:
I just created a copy of upperilm.in to upperilm_ppc64le.in and removed the instructions that were causing the build fail.   Please do a diff between these two files to see what was removed.